ui/app/models/transit-key.js

```diff
@@ -56,13 +80,19 @@ export default DS.Model.extend({
   },
 
   supportedActions: computed('type', function() {
-    return Object.keys(ACTION_VALUES).filter(name => {
-      const isSupported = ACTION_VALUES[name];
-      if (typeof isSupported === 'boolean') {
-        return isSupported;
+    let actions = [];
+    Object.keys(ACTION_VALUES).filter(name => {
+      const keyAction = ACTION_VALUES[name];
+      const isSupported = keyAction.isSupported;
+      if (typeof isSupported === 'boolean' || get(this, isSupported)) {
+        return actions.push({
```

Comment:
rather than pushing objects within a filter (which, when used right, should pair down the array to just the ones you want) maybe you could link a filter and map, like so: 
```const actions = Object.keys(ACTION_VALUES)
    .filter(name => {
        const { isSupported } = ACTION_VALUES[name];
        // return truthy value to include it on the array, or falsy to exclude
        return (typeof isSupported === 'boolean' || get(this, isSupported))
    })
    .map(name => {
        const { description, glyph } = ACTION_VALUES[name];
        return { name, description, glyph };
    })```

Splitting it up this way uses `.filter` as intended, and separates concerns between filtering and data formatting